design_drafts/hardware_status.md

```diff
@@ -45,7 +45,7 @@ string           hardware_id  # unique per‐instance, e.g. "left_wheel/driver"
 
 # ——— Health & Error ——————————————————————————————————————————————
 uint8  health_status         # see HealthStatus enum
-uint8  error_domain          # see ErrorDomain enum
+uint8[]  error_domain        # Array of device errors, because hardware can throw more than one, see ErrorDomain enum
```

Comment:
@MarqRazz I'm still getting together my ideas in a different one. How about something like this?
https://github.com/saikishor/roadmap/commit/db470af4990f33475243bd64d3a66c722222f8f7